index.html

```diff
@@ -40,6 +40,183 @@ <h1>I'm Obedi<br> Glad to see you!</h1>
             </ul>
         </div>
     </section>
+    <section class="card_section">
+        <div class="cards_container">
+            <div class="card_img">
+                <img src="./images/SnapshootPortfolio.svg" alt="card image">
+            </div>
+            <div class="card_text">
+                <h1>Tonic</h1>
+                <div class="card_text_flex">
+                    <p><strong>CANOPY</strong></p>
+                    <img src="images/Vector.png" alt="vector">
+                    <p>Band End Dev</p>
+                    <img src="images/Vector.png" alt="vector">
+                    <p>2015</p>
+                </div>
+                <p class="card_paragraph">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div class="card_lang_flex">
+                    <p><strong>html</strong></p>
+                    <p><strong>css</strong></p>
+                    <p><strong>javascript</strong></p>
+                </div>
+                <div class="card_button">
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+        </div>
```

Comment:
Please ensure that you stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/160081711-462ec23b-34fb-4bce-acd3-67a4c176543a.png) | ![image](https://user-images.githubusercontent.com/10332499/160081630-da41283a-948c-43d1-9381-fd3c08b7c208.png)